tests/plugins/inlinetoolbar/positioning.js

```diff
@@ -0,0 +1,78 @@
+/* bender-tags: inlinetoolbar */
+/* bender-ckeditor-plugins: toolbar,link,inlinetoolbar */
+/* bender-include: ../balloonpanel/_helpers/tools.js */
+/* global balloonTestsTools */
+
+( function() {
+	'use strict';
+
+	var restore = CKEDITOR.dom.window.prototype.getViewPaneSize;
+	CKEDITOR.dom.window.prototype.getViewPaneSize = function() {
+		return {
+			width: 1183,
+			height: 1818
+		};
+	};
+
+	bender.editor = {
+		name: 'editor1',
+		config: {
+			height: 300,
+			width: 300
+		}
+	};
+
+	var spy = sinon.spy,
+		tests = {
+			// There's no need to call editor.window.scrollTo, because balloon logic initially considers only
+			// scrolling of **outer window**. It does not care about editable window, as its scrolling is already
+			// "contained" in result returned by the element.getClientRect() of marker.
+			_getFrameMethodReplaced: false,
+
+			setUp: function() {
+				this.inlineToolbar = new CKEDITOR.ui.inlineToolbar( this.editor, {
+					width: 100,
+					height: 200
+				} );
+
+				this.markerElement = this.editor.editable().findOne( '#marker' );
+
+				// Make sure that global window is scrolled down to 0,0.
+				window.scrollTo( 0, 0 );
+
+				if ( !this._getFrameMethodReplaced ) {
+					sinon.stub( this.editor.window, 'getFrame', { height: 300, width: 300, left: 1, bottom: 643, right: 301, top: 343 } );
```

Comment:
Same here, you can just create stub in `setUp` and restore in `tearDown` - no flag is needed then.